MAINTAINERS.md

```diff
@@ -0,0 +1,8 @@
+The current Maintainers Group for the KubeArchive Project consists of:
+
+* [Andrea Fernandez](https://github.com/mafh314), [Red Hat](https://www.redhat.com)
```

Comment:
Maybe Andrea Fernández (with the accent) ?
@mafh314 what do you think?
```suggestion
* [Andrea Fernández](https://github.com/mafh314), [Red Hat](https://www.redhat.com)
```

Reply:
yes, please :)

Reply:
Added!